docs/community-toolkit/hosting-java.md

```diff
@@ -65,32 +86,41 @@ The following sections detail various example usage scenarios, from hosting a co
 
 ### [Container hosting](#tab/container-hosting)
 
-In the _:::no-loc text="Program.cs":::_file of your app host project, call the `AddSpringApp` method to define the containerized Spring app. Use the `JavaAppContainerResourceOptions` to define the containerized Spring app.
+In the _:::no-loc text="Program.cs":::_ file of your app host project, call the `AddSpringApp` method to define the containerized Spring app. The `JavaAppContainerResourceOptions` allows you to specify the container image and OpenTelemetry agent configuration.
 
 ```csharp
 var containerapp = builder.AddSpringApp(
     "containerapp",
     new JavaAppContainerResourceOptions
     {
-        ContainerImageName = "<repository>/<image>",
-        OtelAgentPath = "<agent-path>"
+        ContainerImageName = "your-registry/your-spring-app:latest",
+        OtelAgentPath = "/agents"
```

Comment:
I would change `/agents` to something else. This has a higher chance of being a real path for someone because it starts with a `/`.